customcontext/texture/eglgralloctexture.cpp

```diff
@@ -148,7 +148,7 @@ NativeBuffer::NativeBuffer(const QImage &image)
     const int w = image.width();
     if (simpleCopy) {
         if (dbpl == image.bytesPerLine()) {
-            memcpy(data, image.constBits(), image.byteCount());
+            memcpy(data, image.constBits(), image.sizeInBytes());
```

Comment:
"This function was introduced in Qt 5.10."

Reply:
Fixed with https://github.com/sailfishos/qtscenegraph-adaptation/pull/1/commits/bb1d0094d4710788fe3c78f6a103f7d237c04967